flink-connectors/flink-connector-kafka/pom.xml

```diff
@@ -78,6 +78,18 @@ under the License.
 			<version>${kafka.version}</version>
 		</dependency>
 
+		<dependency>
+			<groupId>org.apache.kafka</groupId>
+			<artifactId>kafka_${scala.binary.version}</artifactId>
+			<version>${kafka.version}</version>
+			<exclusions>
```

Comment:
Unfortunately, I can't exclude all transitive dependencies because the old tests are instantiating a KafkaServer. We can either think of moving the KafkaSink to a different module or keep it as is until we remove all the old tests and can properly exclude the transitive dependencies.